package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
What's this file for?

Reply:
That file is normally added when using node modules, I think it was added. I will delete it as the codes don't need it